src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -93,6 +93,35 @@
  *  Inet4Address.ofLiteral("02130706689"); // ==> /127.0.1.1
  * }
  *
+ * <p> The above forms adhere "strict" decimal-only syntax.
+ * Additionally, the following (loose) syntax is available with
+ * {@link Inet4Address#ofPosixLiteral(String)} method that implements
+ * <a href="https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_addr.html">
+ * POSIX {@code inet_addr}</a> compatible parsing algorithm, allowing
+ * octal and hexadecimal address segments. Please refer to
+ * <a href="https://www.ietf.org/rfc/rfc6943.html#section-3.1.1"> <i>RFC&nbsp;
+ * 6943: Issues in Identifier Comparison for Security Purposes</i></a>.
```

Comment:
```suggestion
 * <p> The above forms adhere "strict" decimal-only syntax.
 * Additionally, the
 * {@link Inet4Address#ofPosixLiteral(String)} method implements a
 * <a href="https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_addr.html">
 * POSIX {@code inet_addr}</a> compatible "loose" parsing algorithm, allowing
 * octal and hexadecimal address segments. Please refer to
 * <a href="https://www.ietf.org/rfc/rfc6943.html#section-3.1.1"> <i>RFC&nbsp;
 * 6943: Issues in Identifier Comparison for Security Purposes</i></a>.
```

Reply:
@sercher the word "loose" is still there - it's not removed?

Reply:
Right. Apologies, i didn't notice it.